homeassistant/components/reolink/services.yaml

```diff
@@ -16,3 +16,29 @@ ptz_move:
           min: 1
           max: 64
           step: 1
+
+chime_play:
+  target:
```

Comment:
This shouldn't be target. Just add a selector for `device_id`.

Reply:
Sure, that makes more sense. Then the user also can't select entities.

Although I like the UI style of the target more than the select drop-down.

Would be cool if you could just use the target and then somehow specify that only targetting entities is possible. Would make the UI of actions/services more uniform in my opinion.

But anyway that is not something for a integration but for a arch discussion.

Reply:
Done